test/qa/lib/meshnet.py

```diff
@@ -22,16 +30,35 @@ def get_peer_name(output: str, name_type: PeerName) -> str:
         case PeerName.Pubkey:
             return get_this_device_pubkey(output)
 
-def add_peer(ssh_client: ssh.Ssh, tester_allow_fileshare: bool = True, peer_allow_fileshare: bool = True):
+def add_peer(ssh_client: ssh.Ssh,
+             tester_allow_fileshare: bool = True,
+             tester_allow_routing: bool = True,
+             tester_allow_local: bool = True,
+             tester_allow_incoming: bool = True,
+             peer_allow_fileshare: bool = True,
+             peer_allow_routing: bool = True,
+             peer_allow_local: bool = True,
+             peer_allow_incoming: bool = True):
     """
     adds QA peer to meshnet
     try to minimize usage of this, because there's a weekly invite limit
     """
-    tester_allow_fileshare_arg = f"-allow-peer-send-files={str(tester_allow_fileshare).lower()}"
-    peer_allow_fileshare_arg = f"-allow-peer-send-files={str(peer_allow_fileshare).lower()}"
-    sh.nordvpn.mesh.inv.send("--allow-incoming-traffic=true", "--allow-traffic-routing=true", tester_allow_fileshare_arg, PEER_USERNAME)
+    tester_allow_fileshare_arg = f"--allow-peer-send-files={str(tester_allow_fileshare).lower()}"
+    tester_allow_routing_arg = f"--allow-traffic-routing={str(tester_allow_routing).lower()}"
+    tester_allow_local_arg = f"--allow-local-network-access={str(tester_allow_local).lower()}"
+    tester_allow_incoming_arg = f"--allow-incoming-traffic={str(tester_allow_incoming).lower()}"
+
+
+    peer_allow_fileshare_arg = f"--allow-peer-send-files={str(peer_allow_fileshare).lower()}"
+    peer_allow_routing_arg = f"--allow-traffic-routing={str(peer_allow_routing).lower()}"
+    peer_allow_local_arg = f"--allow-local-network-access={str(peer_allow_local).lower()}"
+    peer_allow_incoming_arg = f"--allow-incoming-traffic={str(peer_allow_incoming).lower()}"
+
+    subprocess.run(["nordvpn", "mesh", "inv", "list"], capture_output=True)
```

Comment:
Just do Ctrl+f subprocess in this page :D There are also some lingering imports left.